integration/testdata/run-test/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.15 as builder
```

Comment:
I remember https://github.com/GoogleContainerTools/skaffold/pull/5533/files added these same files. Can't you run this test against the same directory?

Reply:
#5533 is updating an existing file & testing the test dependencies and this is creating a new file. There are no test dependencies. Added a profile to the existing file.